library/oraclelinux

```diff
@@ -4,10 +4,10 @@ GitCommit: 01a15ec99c7470a3391c691509db1759b41eaf66
 GitFetch: refs/heads/master
 # https://github.com/oracle/container-images/tree/dist-amd64
 amd64-GitFetch: refs/heads/dist-amd64
-amd64-GitCommit: b22bc0aae247d85ec26d2acf9d5e2e801fc4ab22
+amd64-GitCommit: c06255193f1e03df034ca0ae218c080f29326e0a
```

Comment:
Can't seem to find this commit on `dist-amd64`; forget to force push? :wink:

Reply:
Whoops. I was testing out some GitHub stuff and forgot to switch remotes. My bad! It should be available now.